pydantic_models/Treatment.py

```diff
@@ -0,0 +1,190 @@
+from pydantic import BaseModel, Field, ValidationError, validator, root_validator
+from datetime import date
+from pydantic.typing import Literal
+
+class Treatment(BaseModel):
+    """
+    Data Model definition for Treatment Entity
+
+    """
+
+    donorId: str = Field(
+        description="Foreign key to map donor ID", 
+        examples="DO-31724")
+    
+    treatmentId: str = Field(
+        description="Primary key for this treatment", 
+        examples="TR-7391493")
+    
+    eventId: str = Field(
+        description="Foreign key to map a patient to the event", 
+        examples="EV-20201123")
+
+    treatmentDate: date = Field(
+        description="Date of treatment", 
+        examples="2020-11-04")
+    
+    treatmentName: str = Field(
+        description="Name of the treatment given to the samples i.e. name of the chemical/drug/therapy", 
+        examples="Carboplatin")
+    
+    treatmentResponse: Literal['Partial Response', 'Complete Response', 'Stable Disease', 'Progressive Disease', 'No Response'] = Field(
```

Comment:
I added another field to capture the response criteria used `treatment_response_criteria_method` in the tab. Let's use that to validate the treatmentResponse values. 

if treatment_response_criteria_method == 'eln dohner aml 2017 oncology response criteria':
       checkList = [
            'complete remission',
            'complete remission with incomplete hematologic recovery (cri)',
            'complete remission without minimal residual disease (crmrd-)',
            'hematologic relapse (after crmrd-, cr, cri)',
            'molecular relapse (after crmrd-)',
            'partial remission',
            'progressive disease',
            'stable disease'
          ]
if treatment_response_criteria_method == 'iwg cheson aml 2003 oncology response criteria':
          checkList = [
            'cytogenetic complete remission (crc)',
            'molecular complete remission (crm)',
            'morphologic complete remission',
            'morphologic complete remission with incomplete blood count recovery (cri)',
            'morphologic leukemia-free state',
            'partial remission'
          ]
if treatment_response_criteria_method ==  'irecist':
          checkList = [
            'immune complete response (icr)',
            'immune confirmed progressive disease (icpd)',
            'immune partial response (ipr)',
            'immune stable disease (isd)',
            'immune unconfirmed progressive disease (iupd)'
          ]
if treatment_response_criteria_method ==  'recist':
          checkList = [
            'complete response',
            'no evidence of disease (ned)',
            'partial response',
            'progressive disease',
            'stable disease'
          ]
if treatment_response_criteria_method ==   'response assessment in neuro-oncology (rano)':
checkList = [
            'complete response',
            'minor response',
            'partial response',
            'progressive disease',
            'stable disease'
          ]
if treatment_response_criteria_method ==  'physician assessed response criteria':
checkList = [
            'physician assessed complete response',
            'physician assessed partial response',
            'physician assessed progressive disease',
            'physician assessed stable disease'
         ]